tests/checkings.test.js

```diff
@@ -1,96 +1,75 @@
 import HomePage from "../pages/HomePage";
 import SignIn from "../pages/SignIn";
-import CheckingPage from "../pages/CheckingPage"
+import CheckingPageForm from "../pages/CheckinPageForm"
+import CheckingPageTable from "../pages/CheckingPageTable";
+import CorrectCheckings from "../datos/CorrectCheckings";
+import IncorrectCheckings from "../datos/IncorrectCheckings"
  
-
-describe('Checkings', () => {
-it('Should take you to Checkings form', async ()=> {
-    await SignIn.open('bank')
+beforeEach(function () {
```

Comment:
You need to make this function ayncrhonous, as you are performing asynchronous operations inside it. Turn the callback function itno an async function and add `await` on each line that interacts with web elements somehow, if not, this part is not going to be executed adequately.

```javascript
beforeEach(async () => {
// logic with pontentially await somewhere goes here
}
```